exist-core/pom.xml

```diff
@@ -1045,7 +1045,12 @@ The BaseX Team. The original license statement is also included below.]]></pream
                         <configuration>
                             <forkCount>1C</forkCount>
                             <reuseForks>false</reuseForks>
-
+                            <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
```

Comment:
I don't understand why these changes are needed here or above? I am also concerned that the `log4j.configurationFile` appears to have been lots. Could you explain please?

Reply:
The mentioned line was dropped by accident and I re-added it in the meantime. The other changes should fixate the test environment in order for calls on methods that return the systems default values will return a predictable result independent on the running system as the environment.